bfdd/bfd_packet.c

```diff
@@ -893,10 +901,31 @@ int bp_set_tos(int sd, uint8_t value)
 	return 0;
 }
 
+static bool bp_set_reuse_addr(int sd)
```

Comment:
The name is kinda misleading. It sets REUSEPORT as well, can we separate them?

Reply:
ok